src/index.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable  import/no-cycle */
+
+import './style.css';
+import interactive from './interactiveList.js';
+import clearCompletedTasks from './addRemove.js';
+
+class SingleToDo {
+  constructor(description) {
+    this.description = description;
+    this.completed = false;
+    this.index = 0;
+  }
+}
+class ToDoList {
+  constructor(toDoTasksArray = [], container) {
+    this.toDoTasksArray = toDoTasksArray;
+    this.container = document.querySelector(container);
+  }
+
+  addToDo(todo) {
+    const newToDo = new SingleToDo(todo);
+    this.toDoTasksArray.push(newToDo);
+    // update index
+    this.toDoTasksArray = this.toDoTasksArray.map((todo, index = 1) => {
```

Comment:
Here you did great work, but maybe you can put this block of code inside a function because you have repeated it inside `removeToDo` method, to not violate the DRY principle.